inst/include/population_dynamics/population/population.hpp

```diff
@@ -33,15 +33,366 @@
 
 #include "../../common/model_object.hpp"
 #include "subpopulation.hpp"
+#include "../recruitment/recruitment.hpp"
+#include "../maturity/maturity.hpp"
+#include "../growth/growth.hpp"
+#include "../fleet/fleet.hpp"
 
 namespace fims {
+  /*TODO: 
+  Review, add functions to evaluate, push vectors back to fleet (or point to fleet directly?)
+*/
+    /**
+     * @brief Population class. Contains subpopulations
+     * that are divided into generic partitions (eg. sex, area).
+     */
+    template <typename Type>
+    struct Population : public FIMSObject<Type> {
+        static uint32_t id_g; /*!< reference id for population object*/
+        size_t nyears; /*!< total number of years in the fishery*/
+        size_t nseasons; /*!< total number of seasons in the fishery*/
+        size_t nages; /*!< total number of ages in the population*/
+        size_t nfleets; /*!< total number of fleets in the fishery*/
+        //constants
+        const double proportion_female = 0.5; /*!< Sex proportion fixed at 50/50 for M1*/
 
-/**
- * Population class. Contains subpopulations
- * that are divided into generic partitions (eg. sex, area).
- */
-template <typename T>
-struct Population : FIMSObject<T> {};
-}  // namespace fims
+        // parameters are estimated; after initialize in create_model, push_back to parameter list - in information.hpp (same for initial F in fleet)
+        std::vector<Type> log_naa; /*!< estimated parameter: log numbers at age*/
+        std::vector<Type> log_Fmort; /*!< estimated parameter: log Fishing mortality*/
```

Comment:
log_Fmort should probably be in the fleet class.

Reply:
So we would set this up with another identical vector in fleet like some of the derived quantities that is held in both the population and fleet classes like expected catch and numbers at age etc.